sqlx-core/src/transaction.rs

```diff
@@ -46,10 +46,10 @@ pub trait TransactionManager {
 /// executed after it was established to be rolled back, restoring the transaction state to
 /// what it was at the time of the savepoint.
 ///
-/// [`Connection::begin`]: struct.Connection.html#method.begin
-/// [`Pool::begin`]: struct.Pool.html#method.begin
-/// [`commit`]: #method.commit
-/// [`rollback`]: #method.rollback
+/// [`Connection::begin`]: crate::connection::Connection::begin()
+/// [`Pool::begin`]: crate::pool::Pool::begin()
```

Comment:
I didn't realize the syntax supported `()` for referring to methods, that's great.